### DISCUSSION TITLE
Get static sandbox without any bundlers in a self hosted bundler?

### DISCUSSION THREAD

**Discussion by ABIBV** [codesandbox/codesandbox-client#6417] [Q&A]
Hi,

I've hosted the sandpack bundler locally for a project.
For the client, I'm using the [@codesanbox/sandpack-client](https://www.npmjs.com/package/@codesandbox/sandpack-client) npm package
I'm trying to create a sandbox with a static template and static environment without any bundlers.
But I get a sandbox with parcel as a bundler everytime.

Here is a sample sandbox [link](https://codesandbox.io/s/h640c) (got from `getCodeSandboxURL()` method) that I've created from my project by passing **static** as the template, but got a parcel sandbox

**Comment by JamesACS** [comment]
Hey @ABIBV - CodeSandbox uses Parcel as it's default bundler - Unfortunately this isn't something you would be able to change unless you were working with Sandpack locally.

**Reply by ABIBV** [reply]
Thank you @JamesACS  for your response.

I've already hosted the sandpack-client locally.
Are you saying like, I've to reconfigure the source of the sanpack then bundle it and host it locally???

Because I just want a sandbox without any bundlers (static template) like [this sandbox](https://codesandbox.io/s/interesting-bird-e8d4g) in which the js files and css files can be linked by the script tag and link tag.

And I saw and option for passing the template in the sandpack-client configuration in the official docs [here](https://sandpack.codesandbox.io/docs/api/client/interfaces/SandboxInfo#template) 
And in the docs of sandpack-react, I saw an parameter for passing the environment [here](https://sandpack.codesandbox.io/docs/api/react/interfaces/SandboxTemplate#environment)

I tried both of them and got nothing but a parcel-bundled sandbox.

**Reply by DeMoorJasper** [reply]
In codesandbox we merge the html entrypoint and sandpack html file so it looks like a static file, but we still run a bundler over it and rehydrate it kind of like how SSR frameworks work. This is currently not possible with the sandpack library.

We also never run any other bundler besides sandpack in the browser, the `parcel` in package.json is just there so you would be able to download the sandbox and run it locally with the same/very similar output.

Not sure if this answers your question?

**Reply by ABIBV** [reply]
@DeMoorJasper 
Thank you for your response.
As you've already mentioned, it doesn't answer my question 😅
But I think, it'll help me in some other way. 🙂

As like I've mentioned in the previous reply, I just want a sandbox in which I can able to link js and css files in the header itself by using script tag and link tag
